domain/infosync/info.go

```diff
@@ -684,13 +687,17 @@ func getServerInfo(id string) *ServerInfo {
 		Lease:          cfg.Lease,
 		BinlogStatus:   binloginfo.GetStatus().String(),
 		StartTimestamp: time.Now().Unix(),
+		Labels:         cfg.Labels,
```

Comment:
Could you add some intergration tests like `select * from information_schema.cluster_info;`

Reply:
updated. PTAL @SunRunAway